cmd/geth/logtestcmd_active.go

```diff
@@ -130,5 +130,9 @@ func logTest(ctx *cli.Context) error {
 		log.Info("Inserted known block", "number", 99, "hash", common.HexToHash("0x12322"), "txs", 10, "gas", 1, "other", "third")
 		log.Warn("Inserted known block", "number", 1_012, "hash", common.HexToHash("0x1234"), "txs", 200, "gas", 99, "other", "fourth")
 	}
+	{
+		log.Info("nil-big.Int", "<nil>", (*big.Int)(nil))
+		log.Info("nil-uint256.Int", "<nil>", (*uint256.Int)(nil))
+	}
```

Comment:
it's better if we commit this to master first, see https://github.com/ethereum/go-ethereum/pull/28501